main.cpp

```diff
@@ -182,75 +202,92 @@ struct RecordingStudio
+        float caclulateTotalFee( float numberOfActualHours = 12.0f, float overtimeRate = 650.50f, float reelsUsed = 4.0f);
     };
 
-    void beginRecordingSession (ControlRoom controlRoomA);
-
-    void startRateClock (bool hasGroupon = true);
+    void beginRecordingSession( ControlRoom controlRoomA );
+    void startRateClock( bool hasGroupon = true);
+    void closeRoom( ControlRoom controlRoomA, bool equipmentOff = true);
+    int billClient( ControlRoom controlRoomA, int hoursUsed = 12, int tapesUsed = 3, int woofersBlown = 3);
 
-    void closeRoom (ControlRoom controlRoomA, bool equipmentOff = true);
-
-    int billClient (ControlRoom controlRoomA, int hoursUsed = 12, int tapesUsed = 3, int woofersBlown = 3);
-
-    //3 things it can do:
-    // record audio
     void recordAudio();
-
-    // send invoices to clients
     void sendInvoice();
-
-    // host events
     void hostEvent();
 };
-/*
 
-Thing 3) Audio Interface
+void RecordingStudio::ControlRoom::bookRoom( bool isRoomBooked, int numberOfTotalClients)
+{
+    if( isRoomBooked == true )
+    {
+        numberOfTotalClients = 2;
+    }
+    else
+    {
+        numberOfTotalClients = 0;
+    }
+}
 
-*/
-struct AudioInterface
+void RecordingStudio::ControlRoom::prepareRoom( bool powerOn, bool roomCleaned, int numberOfAssistants, float rateForAssistant)
 {
+    if( roomCleaned == true )
+    {
+        powerOn = true;
+        rateForAssistant = 1.1f;
+    }
+    else
+    {
+        numberOfAssistants = 1;
+    }
+}
+
+float RecordingStudio::ControlRoom::caclulateTotalFee( float numberOfActualHours, float overtimeRate, float reelsUsed)
+{
+    if( numberOfActualHours >= 8 )
+    {
+        overtimeRate = 2.0f;
+    }
+    else
+    {
+        reelsUsed = 2;
+    }
+        {
+        return ( overtimeRate * numberOfActualHours) + reelsUsed;
+        }
+}
+void RecordingStudio::recordAudio(){}
+void RecordingStudio::sendInvoice(){}
+void RecordingStudio::hostEvent(){}
 
-    //number of inputs (int)
+
+struct AudioInterface
+{
     int audioInputs = 16;
-    //types of inputs (std::string)
     std::string inputType = ("XLR" "TRS" "MIDI" "Optical");
-    //types of enclosure material (std::string)
     std::string enclosureMaterial = ("Metal" "Plastic");
-    //types of outputs (std::string)
     std::string outputType = ("XLR" "TRS" "MIDI" "Optical" "RCA");
-    //locations of inputs (std::string)
     std::string inputsLocation = ("Front" "Back");
 
-    //3 things it can do:
-    // receive audio signals
     void receiveAudio();
-
-    // display signal levels
-    float displayLevels(); //return is displayed for user
-
-    // output audio signals
+    float displayLevels();
     void outputAudio();
 };
-/*
 
-Thing 4) Stereo Widener Audio Plugin
+void AudioInterface::receiveAudio(){}
+float AudioInterface::displayLevels()
+    {
+    return 2.2f;
+    }
```

Comment:
Fix this formatting
- [ ] revise

Reply:
Fixed, based on this line in your SimpleEQ project: `juce::Path getPath() { return leftChannelFFTPath; }`